### DISCUSSION TITLE
Write throughput very slow after 10-20 seconds

### DISCUSSION THREAD

**Discussion by queen4me** [trapexit/mergerfs#1297] [General]
I just configured a new mergerfs pool consisting of 3x8TB Seagate HDD and pfrd policy.
Having them attached in a pc the throughput when copying from one to the other was 115.000 kbytes/sec from beginning to end of the copy.
Using the same drives with mergerfs starts with 115.000 kbytes/sec but gets very slow after 10-20 seconds writing.
The pool is shared via SMB/CIFS.
Does someone have an explanation for this behaviour and is there any remedy/solution?

Thanks a lot for helping.

**Comment by trapexit** [comment]
You aren't providing sufficient info to comment. What are the settings? What is "slow"? What is the load of the host? The RAM usage? etc. Please provide the details requested by the ticket and in the docs: https://github.com/trapexit/mergerfs?tab=readme-ov-file#support

**Reply by queen4me** [reply]
Thanks trapexit for your quick answer. 
Should I first give it another try with the same NAS (OMV) but without mergerfs?

**Reply by trapexit** [reply]
Give what a try? 

As I say above and in the docs I need all the information you can provide me to most offer the best feedback so do whatever you need to to provide that information.

**Reply by queen4me** [reply]
I'll give you all informations that I can provide because I'm very interested using mergerFS because this seems to be the perfect solution for my use case.
My first idea after your reply was to share SMB/CIFS without mergerFS with the same hardware OMV 6 ist installed.
But you're probably right that we solve the issue best and fast when I provide you with all the information possible:

My NAS consists of a motherboard with Intel i7-4770 CPU and 16MB RAM using OMV 6.9.12-2 with OMV-mergerfs Addon 6.3.8.
Attached to the onboard SATA-Controller are 3 Seagate ST8000DM004 drives with 8TB per drive.
The drives are formatted with EXT4 and I have one mergerFS pool consisting of 3x8=24TB.
The NAS ist connected to a 24-port GBit-Switch from HP ony which two NAS-systems using OMV 3 and 4 are also attached. With both systems I have about 100.000-110.000 kbytes/sec when writing to the RAID-5 shares from an Windows 10 client attached to my LAN.

**Reply by trapexit** [reply]
The details I need are as described in the docs I shared.
